includes/ApplHandlingByStr.php

```diff
@@ -82,6 +86,22 @@ function showFullStatus( $stat_str )
 }
 
 
+function findPending( $stat_str )
+{
+	$STR = explode ( ';', $stat_str );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->